src/cp2k_info.F

```diff
@@ -274,6 +274,15 @@ FUNCTION cp2k_flags() RESULT(flags)
       flags = TRIM(flags)//" greenx"
 #endif
 
+! Preprocessor macros used in openPMD.cpp:
```

Comment:
Sorry, I forgot about this. We usually add exception [here](https://github.com/cp2k/cp2k/blob/f0e2e694aacf72b5fbb4a35afc4edb64dda8b7ec/tools/precommit/check_file_properties.py#L20), but your solution also gets the job done ;-)